### PR TITLE
5982 edit group name and color

### PR DESCRIPTION
**Pull request by endulab**
### What does the PR do

The user is able to change group chat's color. Changing image will be added in the next phase.
Additionally, users have got the same chat color during creation.
Fix added - changes can be done only by the admin.

Required  prs:
- status-go - https://github.com/status-im/status-go/pull/2754
- StatusQ - https://github.com/status-im/StatusQ/pull/777

### Affected areas

Chat

### Screenshot of functionality (including design for comparison)

https://user-images.githubusercontent.com/61889657/179198071-f2109c9c-ab58-45c2-94be-f4dd6a477652.mov

### CODE REVIEW

File: src/app/modules/main/chat_section/module.nim [hunk 1]
```diff
@@ -147,7 +147,12 @@ proc buildChatSectionUI(
       colorHash = self.controller.getColorHash(chatDto.id)
       colorId = self.controller.getColorId(chatDto.id)
 
-    let amIChatAdmin = (self.amIMarkedAsAdminUser(chatDto.members) or channelGroup.admin)
+    # for group chats only member.admin should be checked,
+    # because channelGroup.admin is alway true
```
Comment (jrainville) [inline src/app/modules/main/chat_section/module.nim:151]: this might be a bug on status-go

**Review by jrainville — APPROVED** [review]
Great work!
Don't forget to assign a QA to this PR

**Review by elina2015 — APPROVED** [review]
@endulab thank you for the great implementation of the new functionality! There is only one small thing - the space after the standard colors is smaller than on the Design. Could you kindly check?:
image

**Review by stefandunca — APPROVED** [review]
It looks good. I also tested the functionality, besides the missing image, everything works as expected.

File: src/app/modules/main/chat_section/module.nim (hunk 1 above)
Comment (endulab) [reply src/app/modules/main/chat_section/module.nim:151]: Hmm. Channel group contains a lot of chats. There can be 1:1 chats, group chats, global chats. This parameter makes sense only for 1:1 chat and community groups.
Maybe the better approach would be to move admin flag to chat structure rather then having it for a group.

**Comment by endulab** [comment]
> @endulab thank you for the great implementation of the new functionality! There is only one small thing - the space after the standard colors is smaller than on the Design. Could you kindly check?: image

@elina2015 new build should have a fix.

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Pull request merged**